live-build/variants/internal-dcenter/ansible/playbook.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- hosts: all
+  connection: chroot
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: /usr/bin/python2
+  roles:
+    - appliance-build.minimal-common
+    - appliance-build.minimal-internal
+    - appliance-build.install-delphix-zfs-packages
+    - appliance-build.zfsonlinux-development
+    - appliance-build.masking-common
+    - appliance-build.masking-development
```

Comment:
For my own education, why do we need the `appliance-build.masking-development` role? Same comment applies to `appliance-build.virtualization-development`? If we can eliminate these roles, that could reduce the size of the image, which would be a net gain in my opinion.